test/capability-arbitrary-precision-integers.ll

```diff
@@ -50,13 +50,13 @@ entry:
   store i30 1, i30* %a.addr, align 4
 ; CHECK-LLVM: store i48 -4294901761, i48 addrspace(1)* @c
   store i48 -4294901761, i48 addrspace(1)* @c, align 8
-  store i96 1, i96 addrspace(1)* @d, align 8
-; CHECK-LLVM: store i96 1, i96 addrspace(1)* @d
+  store i96 18446744073709551620, i96 addrspace(1)* @d, align 8
```

Comment:
Is there present check of how this value is stored in SPIR-V file?

Reply:
Yes, line 22: ; CHECK-SPIRV-DAG: Constant [[#I96]] [[#]] 4 0 1

NOTE: 18446744073709551620 = 2^64 + 4

Reply:
I got it, thank you.